CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 ## XX.XX.XX - 2023-XX-XX
 
 ### PaymentSheet
-* [ADDED] PaymentSheet now supports the following payment methods for SetupIntents and PaymentIntents with setup for future usage:
```

Comment:
I don't think we want this change.

Reply:
Well, I added this because the statement below wasn't true, alipay and OXXO are not for PI and SI, only for PI, so I moved them into a different section

Reply:
Ahh, my bad! I didn't read the new section part. Good call.